linux/keyman-config/keyman_config/kvk2ldml.py

```diff
@@ -349,7 +364,18 @@ def convert_ldml(kvkData):
             else:
                 keymaps["shift"] = (uskey,)
 
+    info, system, options, keyboards, files = parsemetadata(kmpJsonFilename)
+
     ldml = etree.Element("keyboard", locale="zzz-keyman")
+    for keyboard in keyboards:
+        if keyboard['id'] != keyboardName:
+            continue
+        if 'oskFont' in keyboard:
+            fontFile = os.path.join(os.path.dirname(kmpJsonFilename), keyboard['oskFont'])
+            font = _fontFacename(fontFile)
```

Comment:
What happens if this returns empty string?

Reply:
Done.